README.md

```diff
@@ -56,6 +56,25 @@ vars:
 
 > Note: `sprint` and `status` will always be tracked, as they are necessary for creating common agile reports. 
 
+
+### Disabling models
+
+It's possible that your Jira connector does not sync every table that this package expects. If your syncs exclude certain tables, it is because you either don't use that functionality in Jira or actively excluded some tables from your syncs. To disable the corresponding functionality in the package, you must add the relevant variables. By default, all variables are assumed to be `true`. Add variables for only the tables you would like to disable:  
+
+```yml
+# dbt_project.yml
+
+...
+config-version: 2
+
+vars:
+  jira_source:
+    using_sprints: false # Disable if you do not have the sprint table, or if you do not want sprint related metrics reported
+  jira:
+    using_sprints: false # Disable if you do not have the sprint table, or if you do not want sprint related metrics reported
```

Comment:
do users need to set this variable in both packages? would it be better to make it a global variable (with a more specific name like `jira_using_sprints`) ?

Reply:
also should we note that the sprint table stuff will not impact `sprint` being included in the daily issue field history model? since that model pulls sprints (or rather sprint_id's) from `issue_combine_field_histories` and doesn't actually use the `sprint` table itself

Reply:
Thanks for the suggestion! I've renamed the variable and will make it global.  

If a customer doesn't use sprints, will the values for the daily issue field history models just be null, or will users run into an error?